src/patternfly/utilities/Width/width.scss

```diff
@@ -0,0 +1,15 @@
+@import "../../patternfly-utilities";
+
+/* stylelint-disable */
+
+// Width options
+$pf-u-width-options: (
+  width-auto:    (width auto),
+  width-full:    (width 100%),
+  width-initial: (width initial),
```

Comment:
This is coming out as width: auto; @mattnolting / @michael-coker any thoughts on that?

Reply:
the initial value of width is `auto` so I don't think that's necessary. https://developer.mozilla.org/en-US/docs/Web/CSS/width

Reply:
`auto` is now removed. I've added a parenthetical "(auto)" to each example to point out that `-initial` is effectively the same as `auto`.